podvm/Dockerfile.podvm_builder.rhel

```diff
@@ -1,4 +1,3 @@
-# syntax=docker/dockerfile:1.5-labs
```

Comment:
Is there a reason to remove this syntax ?

Reply:
Afaicr, the reason for adding it was to handle buildkit features. If the build works with the syntax intact, I would prefer not to remove it to avoid any surprises.

Reply:
I must have missed it. It is restored now